common/model/pages.ts

```diff
@@ -1,10 +1,13 @@
 import { GenericContentFields } from './generic-content-fields';
 import { Link } from './link';
 import { Season } from './seasons';
+import { Format } from './format';
 
 export type Page = GenericContentFields & {
   type: 'pages';
+  format: Format | null;
```

Comment:
Could we use `undefined` please?

```suggestion
  format: Format | undefined;
```